lte/protos/session_manager.proto

```diff
@@ -800,23 +861,28 @@ message RatSpecificContextAccess {
 //description provided by the UE in an IMS session.
 
 message M5GSMSessionContextAccess{
-    //One byte, received from UE. For 5G pdu_session_id.
-    bytes pdu_session_id = 1;
+    //One byte, received from UE. For 5G pdu_session_id. 
+    uint32 pdu_session_id = 1;
     //selected PDU session TS-24501 - 9.11.4.11
     PduSessionType    pdu_session_type = 2;
     //selected session and service continuty mode TS-24501 - 9.11.4.16
     SscMode selected_ssc_mode = 3;
     //TS-24501 - 9.11.4.3 - minimum 7 and max 65538, variable length
     repeated QosRules authorized_qos_rules = 4;
     //TS-24501 - 9.11.4.14 PDU session Aggregate Maximum Bit Rate imported from subscriberdb.proto
-    AggregatedMaximumBitrate session_ambr = 5;
+    //    AggregatedMaximumBitrate session_ambr = 5;
+    SessionAmbr  session_ambr = 5;
```

Comment:
Why is this needed? No need to define another message type for the same purpose of AggregatedMaximumBitrate.

Reply:
It was commented anway,  Now I removed the same.

Reply:
I suggest that you remove SessionAmbr completely not the other way around.